src/main/resources/BridgeServer2.conf

```diff
@@ -176,3 +176,7 @@ api.developer.password = dummy-value
 
 shared.developer.email = dummy-value
 shared.developer.password = dummy-value
+
+synapse.oauth.url = https://repo-prod.prod.sagebase.org/auth/v1/oauth2/token
+synapse.oauth.client.id = dummy-value
+synapse.oauth.client.secret = dummy-value
```

Comment:
Where do we get these values so that I can (a) configure my local server and (b) so we can add these to the BridgeServer2-infra project?

Reply:
Explanation of these values added to the Bridge bootstrap document, the values are checked into LastPass, and I've created a PR to add the values for Elastic Beanstalk deployments.